src/fides/api/ctl/migrations/versions/c9ee230fa6da_add_config_set_column_to_application_.py

```diff
@@ -22,10 +24,28 @@ def upgrade():
         sa.Column(
             "config_set",
             sqlalchemy_utils.types.encrypted.encrypted_type.StringEncryptedType(),
-            nullable=False,
+            nullable=True,  # keep it nullable to start, in case we have any existing records
         ),
     )
 
+    # Data migration - delete the existing app config record, if it exists
+    # since we're making a new encrypted JSON column, we can't reasonably set a default
+    # so we must just delete and start over. up to this point, users could only have
+    # the storage.active_default_storage_type property set in this record. they will have
+    # to reset it after this migration
+    bind = op.get_bind()
+    existing_appconfig: TextClause = bind.execute(
```

Comment:
eh, this type looks like it's a ResultProxy instead of a TextClause, but I've made that type mistake on earlier migrations!

Reply:
good catch - will no longer be relevant in my new commit though, so resolving :)